doc/base/classes.xml

```diff
@@ -20124,12 +20184,32 @@
 			</argument>
 			<description>
 				Set the maximum amount of characters the [LineEdit] can edit, and cropping existing text in case it exceeds that limit. Setting 0 removes the limit.
+        
+        [code]
+        extends Node2D
+        
+        onready var __input = find_node("LineEdit")
+        
+        func _ready():
+          __input.set_max_length(140) # set input to max 140 characters for a tweet
```

Comment:
I don't think such examples are useful at all, it's pretty obvious from the function definition that it's how they're meant to be used. If we start adding code snippets with
```
extends Node2D

onready var somestuff

func _ready():
    use_function_described_above(some_arg)  # do what the function says it does
```
we're not really teaching people anything meaningful, and we're bloating the docs with lots of noise.